### DISCUSSION TITLE
After the student package authentication is passed,  Copilot cannot be used？

### DISCUSSION THREAD

**Discussion by 18801179084** [community/community#119863] [GitHub Education]
### Hi everyone,

After the student package authentication is passed, the account has been marked with Highlights purple, but Copilot cannot be used. How can we solve this problem?
![image](https://github.com/community/community/assets/49431018/605f0c8a-a791-4c66-b8ae-7256d2b78855)
![image](https://github.com/community/community/assets/49431018/65605268-282f-4ca0-a374-fedcc8e9ec94)
![image](https://github.com/community/community/assets/49431018/c2b779e8-6d6c-485c-b638-8d0c2b1cae00)

**Comment by ScripterDude** [comment]
same issue

**Comment by gurjeetsinghvirdee** [comment]
Hey @18801179084 
Install co-pilot extension in your IDE, It will work

**Reply by Ghostjing** [reply]
It did not work after I installed it.

**Reply by 18801179084** [reply]
Install the extension in vscode, and after logging in to the github account, it prompts an invalid copilot token; 403

**Reply by PythonSmall-Q** [reply]
with me, it is r.response is not a function and it won;t work nomatter what i do

**Comment by 18801179084** [comment]
Has anyone solved this problem?

**Reply by KenjiOhtsuka** [reply]
@18801179084 
Hello, I encountered the same problem.
But I could get it activated via "Your Copilot" menu item.

![image](https://github.com/community/community/assets/1572810/4bd54eb1-1452-43e0-ab91-2c0bc5946e9d)

**Reply by KenjiOhtsuka** [reply]
After that, I restarted my IDE, and the function worked.

**Comment by 18801179084** [comment]
![image](https://github.com/user-attachments/assets/a5243438-1808-4baf-ac73-64d90cc8722e)
why????

**Comment by 18801179084** [comment]
i can't use copilot in vscode!!!

**Comment by 18801179084** [comment]
Do I need to modify any configuration items in my account?
